content/en/docs/topics/permissions_sql_storage_backend.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Permissions management for SQL storage backend"
+description: "Get to know how to setup permissions when using SQL storage backend."
+aliases: ["/docs/permissions_sql_storage_backend/"]
+---
+
+This document aims to provide guidance to users for setting up and managing permissions when using the SQL storage backend.
+
+## Introduction
+
+To handle permissions, Helm leverages the RBAC feature of Kubernetes. When using the SQL storage backend, Kubernetes' roles can't be used to determine whether or not an user can access a given resource. This document shows how to create.
```

Comment:
This document shows how to create. --> This document shows how to create and manage these permissions.